extensions/amp-viewer-integration/0.1/test/test-touch-handler.js

```diff
@@ -114,21 +114,59 @@ describes.fakeWin('TouchHandler', {}, (env) => {
       expect(messages[0].data.data.type).to.equal('touchstart');
     });
 
-    it('should not forward events marked with `shouldViewerCancelPropagation`', () => {
-      const event = fakeTouchEvent('touchmove');
-      event.stopImmediatePropagation = env.sandbox.spy();
+    describe('event propagation', () => {
+      let event, event2;
+
+      beforeEach(() => {
+        event = fakeTouchEvent('touchmove');
+        event2 = fakeTouchEvent('touchmove');
+        event.stopImmediatePropagation = env.sandbox.spy();
```

Comment:
Why isn't `stopImmediatePropagation` included as a property in the `fakeTouchEvent`?

Reply:
I can move it.